.gitmodules

```
@@ -122,3 +122,6 @@
 [submodule "addons/minetest/module"]
 	path = addons/minetest/module
 	url = https://git.sr.ht/~fgaz/minetest-lls-definitions
+[submodule "addons/REFramework-LLS/module"]
+	path = addons/REFramework-LLS/module
+	url = https://github.com/youwereeatenbyalid/REFramework-LLS
```

The `url` should be a git clone URL, not the HTML page, like the below:
```suggestion
	url = https://github.com/youwereeatenbyalid/REFramework-LLS.git
```

Did you add the submodule using the following command?
`git submodule add https://github.com/youwereeatenbyalid/REFramework-LLS.git addons/REFramework-LLS/module`

---

I did not, wrote  `git submodule add https://github.com/youwereeatenbyalid/REFramework-LLS addons/REFramework-LLS/module` I'm pretty certain. Didn't realize there was a difference between the two.